Character Thought; Joycean Compounds

sk3853 opened the pull request:
Identifies regions of characters' streams-of-consciousness; identifies Joycean compound words with `<seg type="neologism">`.

sk3853 committed:
Character Thought Telemachus

Identifies regions of Stephen's stream-of-consciousness throughout Telemachus

sk3853 committed:
Character Thought Nestor

Identifies regions of Stephen's stream-of-consciousness throughout Nestor

sk3853 committed:
Character Thought Proteus

Identifies regions of Stephen's stream-of-consciousness throughout Proteus
NOTE: Within this episode is a full scene that occurs within Stephen's thoughts (Stephen imagines what it would be like if he entered his Aunt Sara's house). I have annotated this region as though it actually happened, rather than identifying the entire scene as one block of Stephen's thought.

sk3853 committed:
Character Thought/Joycean Compounds Calypso

Identifies regions of Mr Bloom's stream-of-consciousness throughout Calypso; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Character Thought/Joycean Compounds Lotus Eaters

Identifies regions of Mr Bloom's stream-of-consciousness throughout Lotus Eaters; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Character Thought/Joycean Compounds hades

Identifies regions of Mr Bloom's stream-of-consciousness throughout Hades; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Character Thought/Joycean Compounds Aeolus

Identifies regions of Mr Bloom and Stephen's streams-of-consciousness throughout Aeolus; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Character Thought/Joycean Compounds Lestrygonians

Identifies regions of Mr Bloom's streams-of-consciousness throughout Lestrygonians; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Editing neologism

sk3853 committed:
Character Thought/Joycean Compounds Syclla 

Identifies regions of Stephen and Mulligan's streams-of-consciousness throughout Syclla and Charybdis; identifies Joycean compound words with <seg type="neologism">.

sk3853 committed:
Editing Character Thought

Add character thought for Buck Mulligan

JonathanReeve commented:
Thanks for this! Looks great. I'll review this in detail soon. In the meantime, if you want to try changing some of the `<seg type="neologism">` to `<distinct type="X">` (where X is the type of neologism/compound), that'd be awesome. You could even do a global find/replace using a text editor, if you want. But if you don't feel like it, don't worry about it—it's easy for me to change later.

sk3853 committed:
Tidying up <distinct type="X">

sk3853 committed:
Editing <distinct type="X">

sk3853 committed:
Editing <distinct type="X">

sk3853 committed:
Tidying up character thought

sk3853 committed:
Tidying up thought and distinct punctuation

sk3853 committed:
Tidying up thought and distinct punctuation

sk3853 committed:
Swapping neologism for distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Replacing neologism with distinct

sk3853 committed:
Tidying up <p>

sk3853 committed:
Swapping neologism for distinct

sk3853 committed:
Tidying up <said who> and <distinct type>

sk3853 committed:
Swapping neologism for distinct

sk3853 committed:
Swapping neologism for distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up said

sk3853 committed:
Swapping neologism for distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Adding distinct types and character thought

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up distinct

sk3853 committed:
Character thought and neologisms Sirens

listed "mooncarole" as a Joycean because I can't find a source or the meaning of the word, though it could potentially be nonstandard-compound (?)
listed each instance of "anear" as a nonstandard-compound, but I'm wondering if they should be ignored altogether?
unsure if "nakkering" should be archaism or Joycean-- in the OED it's listed as "naker," an "historical" word for a kettledrum; Gifford says it means "to sound a kettledrum," but Joyce repurposes the verb to be used with a different instrument. I left it at archaism for now

sk3853 committed:
Tidying up character thought/neologisms

sk3853 committed:
Tidying

sk3853 committed:
Tidying up distinct

sk3853 committed:
Tidying up character thought

sk3853 committed:
Tidying

sk3853 committed:
Distinct types

Because this whole chapter is in 1st person, I didn't add any character thought annotations. Let me know if there actually do exist some that may have slipped through the cracks.
Unsure about the definition of "pomellated" on line 97, but listed it as Joycean because no results were found online, nor in Gifford.
"Volupcy" on 355: Gifford suggests Joyce coins this word, although jjon.org believes that the word was used a couple times before Ulysses, and Joyce may be repurposing it. For simplicity's sake I'm marking it as Joycean, although it could possibly go unlabeled or listed as an archaism.
Although words like "devanic" (359) and "jivic" (341) don't show up in the OED, Gifford defines them so I'll leave them unlabeled 
"Mimber" on line 825 I think refers to the word "number," as it appears in archaic text (?). I left it unlabeled because I couldn't find any concrete information about it online, but it could maybe be marked as an archaism. 
Added "bretelles" on line 1283 as foreign (fr).

sk3853 committed:
Tidying up Distinct

sk3853 committed:
Character Thought and Distinct Type

"eyebrownleine" was left unlabeled because it is a misspelling of a brand Joyce was making reference to. It could alternately be listed as an archaism. Same goes for "bearsgrease."

JonathanReeve commented:
This looks awesome, Pia. I'll start reviewing this, and invite Ronan to review it, too.

JonathanReeve commented on u01_telemachus.xml:
```diff
@@ -205,14 +205,14 @@
 <lb n="010211"/>functioning. She calls the doctor sir Peter Teazle and picks buttercups off
 <lb n="010212"/>the quilt. Humour her till it's over. You crossed her last wish in death and
-<lb n="010213"/>yet you sulk with me because I don't whinge like some hired mute from
+<lb n="010213"/>yet you sulk with me because I don't <distinct type="dialect">whinge</distinct> like some hired mute from
```
I'm not so sure about this one. "Whinge" seems fairly standard (though informal) British English. But I also feel like erring on the side of more markup is a good idea.

JonathanReeve commented on u03_proteus.xml:
```diff
@@ -1,15 +1,15 @@
-<lb n="030007"/>di color che sanno</quote>. Limit of the diaphane in. Why in? Diaphane,
-<lb n="030008"/>adiaphane. If you can put your five fingers through it it is a gate, if not a
-<lb n="030009"/>door. Shut your eyes and see.</p>
+<lb n="030007"/>di color che sanno</quote>. Limit of the <distinct type="archaism">diaphane</distinct> in. Why in? <distinct type="archaism">Diaphane,</distinct>
```
I'm not sure this is an archaism, necessarily, but rather just a term from Greek philosophy that might not be very common, like "ethos." I'd recommend maybe changing this to `<foreign>`, instead.

JonathanReeve reviewed (approved):
I think this looks great. I looked at the first four chapters, and then skimmed the rest. I made a couple of notes here and there, but I'd recommend that we merge this. I'll check this out on a new branch, fix the validation errors, and look it over one more time. Then I can try to visualize this with XSL, so that it's easier to review.

sk3853 committed:
Distinct types Oxen

"Swire ywimpled" on line 81 is attributed to anglo-saxon and middle english in Gifford, but I left them unlabeled because I don't know if they'd be considered archaism
Unsure about "drad" on line 82
Unsure about "ongot" on line 93
"donought" on line 554 is dialectical, archaic, and a compound word. I marked it as "compound"
Unsure about "hentrusion" on line 1448
Though a lot of the words in the end of this chapter can be found in the OED, such as "bar, wing, twig" etc., I mark them as dialect according to Gifford.
Difficult towards the end to tell which words are dialectical and which words are attempting to get across that the narrator is drunk; I used my judgement but some may be incorrect. For example, 1507 "shee," "shir."

sk3853 committed:
Tidying up distinct

JonathanReeve commented:
I'll just merge this and sort it out later, if necessary.
